src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5256,4 +5256,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="IDS_FeatureStackAllocInitializer" xml:space="preserve">
     <value>stackalloc initializer</value>
   </data>
+  <data name="ERR_InvalidStackAllocArray" xml:space="preserve">
+    <value>"Invalid rank specificier: expected ']'</value>
```

Comment:
>specificier [](start = 25, length = 11)

"specificier"???  I think this is a typo.

Reply:
oops.